### ISSUE TITLE
A11y_AzureDataStudioStayHealthy_Notebook_Markdown Toolbar_MouseHover : Toolbar buttons tooltip is not hoverable.

### ISSUE THREAD

**Issue by pranshu512** [microsoft/azuredatastudio#10654]
“[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

GitHubTags:#A11y_AzureDataStudioStayHealthy;#A11ySev2;#A11yTCS;#A11yWCAG2.1;#MAS1.4.13;#DesktopApp;#SQL Azure Data Studio;#FTP;#Benchmark;#Win32;

**Environment Details:**
Application Name: Azure Data Studio
Version: 1.19.0-insider (system setup)
Commit: 8288360
Date: 2020-05-29T05:54:49.731Z
VS Code: 1.46.0
Electron: 7.2.4
Chrome: 78.0.3904.130
Node.js: 12.8.1
V8: 7.8.279.23-electron.0
OS: Windows_NT x64 10.0.19041

**Additional Details:**
MAS References: MAS1.4.13

**Repro Steps:**

1. Launch Azure Data Studio Insiders application.
2. Connect to server.
3. Navigate to File --> New Notebook.
4. Navigate upto + Cell button in notebook, hit enter & select markdown cell.
5.Hover over toolbar buttons then tooltip will appear.
6.Check whether tooltip is hoverable or not using mouse.

**Actual:**
Toolbar buttons tooltip content is not hoverable using mouse.

**Similar issue also observed in following scenarios:**
1)Notebook-->Toolbar 

**Expected:**
Toolbar buttons tooltip content should be hoverable using mouse.

**User Impact:**
Low vision users will face difficulty to view the tool-tip content. 

**Attachment link for Reference:**

[10654_A11y_AzureDataStudioStayHealthy_Notebook_Markdown Toolbar_MouseHover.zip](https://github.com/microsoft/azuredatastudio/files/4707533/10654_A11y_AzureDataStudioStayHealthy_Notebook_Markdown.Toolbar_MouseHover.zip)

**Comment by lucyzhang929** [comment]
![Screen Shot 2020-07-16 at 2 42 34 PM](https://user-images.githubusercontent.com/23462877/87731085-71f66000-c77e-11ea-97b9-23cad6e3883f.png)

**Comment by hannahqin** [comment]
@pranshu512 can we close this issue based on Alexandria's comment above indicating that this should not be an accessibility issue?

**Comment by alanrenmsft** [comment]
please reopen if you have any questions. @pranshu512
